contracts/rate_oracles/BaseRateOracle.sol

```diff
@@ -309,29 +295,28 @@ abstract contract BaseRateOracle is IRateOracle, Ownable {
         }
     }
 
+    /// @notice Computes the growth between two rates. This should be implemented
+    /// depending on how the rate evolves. If rate compounds, this should be
+    /// a division i.e. (rateTo/rateFrom) - 1. If the rate does not compound,
+    /// it should be a subtraction i.e. rateTo - rateFrom.
+    /// @param rateFromRay Un-annualised starting rate (in ray)
```

Comment:
maybe worth mentioning in the comments that rate actually refers  to a liquidity index like object rather than a rate of return ?

Reply:
[c2a5c2f](https://github.com/Voltz-Protocol/voltz-core/pull/230/commits/c2a5c2fda8ffd7f8ede64ea6b3b1d51a822d32d0)